### PR TITLE
Connecting the help menu for anvi'o programs with online help

### PR DESCRIPTION
**Pull request by meren**
This PR overloads the `argparse` module to add additional information at the end of the help menu for anvi'o programs and scripts IF there is an entry for them in the `docs` module.

This is related to #1556 and the `epiloge` can be extended with additional information.

----

Here is an example for `anvi-profile`:

image

Here is an example for `anvi-script-gen-CPR-classifier` (for which here is no entry in `docs`):

image

You tell me if it is crazy useful or crazy useless.

### CODE REVIEW

**Comment by meren** [comment]
(it learns the version the user is using, and constructs the URL according to that).

File: anvio/argparse.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        return help_text + "\n"
+
+
+    def get_args(self, parser):
+        """A helper function to parse args anvi'o way.
+
+           This function allows us to make sure some ad hoc parameters such as `--debug`
+           can be used with any anvi'o program spontaneously even if they are not explicitly
+           defined as an accepted argument, yet flags (or parameters) anvi'o does not expect
+           to see can still be sorted out.
```
Comment (ekiefl) [inline anvio/argparse.py:112]: Wrong indentation

**Comment by ekiefl** [comment]
This is fantastic. I'm positive people will use this.

**Comment by ekiefl** [comment]
I agree with leaving out resource links. People will click the online help, which has the resource links.

**Review by ivagljiva — APPROVED** [review]
Awesome! This will be crazy useful, Meren :) 🎉

**Comment by meren** [comment]
With the last commit this is how the very end of `anvi-interactive -h` looks like in dark terminal:

image

versus in a light terminal:

image

If a program doesn't have an entry under `anvio/docs/programs` this is how it looks then:

image

**Comment by ivagljiva** [comment]
can I just say that I LOVE how it goes "can consume (CHEESE), can provide (PIZZA), and you can learn more about how it does that with (FIRE)". 

it makes so much sense. Anvi'o is making pizza for me.

**Pull request merged**